react-native/react/common-adapters/input.desktop.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from '../base-react'
+import {TextField} from 'material-ui'
+import {globalStyles} from '../styles/style-guide'
+
+// Floating label isn't very controllable, not going to make this pixel perfect
```

Comment:
This control changes bunch in the next version of the material design ui (still in beta). It's kinda hard to get this to work pixel perfect with the design (and this is a temp design anyways) so let's go with this until we make a decision later on